src/modules/database/database.ts

```diff
@@ -0,0 +1,41 @@
+import mongoose from 'mongoose';
+
+class Database {
+  private static instace: Database | null = null;
```

Comment:
how about changing this variable name to `singleton` to make it more clear to the developer?